### ISSUE TITLE
invoke-as   Illegal instruction

### ISSUE THREAD

**Issue by muziling** [MiCode/legacy-patchrom#10]
Can I found /system/xbin/invoke-as source code and build an version for my phone.

invoke-as had compiled with dyn link, and can not run at my phone:
invoke-as:   ELF 32-bit LSB executable, ARM, version 1 (SYSV), dynamically linked (uses shared libs), stripped

 I/DEBUG   (   89): pid: 415, tid: 415  >>> /system/xbin/invoke-as <<<
 I/DEBUG   (   89): signal 4 (SIGILL), code 1 (ILL_ILLOPC), fault addr 000086b0
 I/DEBUG   (   89):  r0 00000003  r1 be90ac44  r2 be90ac54  r3 4000800c

**Comment by HackerMi** [comment]
Try the "patchrom/miui/system/xbin/invoke-as".

**Comment by muziling** [comment]
Yes, I got invoke-as from patchrom,  maybe compiled for armv7, but I am armv6...    So miui theme not install successed.
